### PR TITLE
Database models

### PR DESCRIPTION
**Pull request by adasikow**
I've created models describing data stored in database. We forgot to discuss about poassible entry categories, statuses and report types (look at TODO in main/models.py)

### CODE REVIEW

File: authentication/models.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class UserProfile(models.Model):
+
```
Comment (artpard) [inline authentication/models.py]: Unnecessary blank line.

File: main/models.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# coding=utf-8
+from django.db import models
+from authentication.models import UserProfile
+
+
+class Entry(models.Model):
+
```
Comment (artpard) [inline main/models.py]: Unnecessary blank line

File: main/models.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    location = models.TextField()
+    category = models.TextField(choices=CATEGORIES)
+    status = models.IntegerField(choices=STATUSES, default=0)
+    score = models.IntegerField(default=0)
+    timestamp = models.DateTimeField(auto_now_add=True)
+    is_archived = models.BooleanField(default=False)
+
+
+class Comment(models.Model):
+
```
Comment (artpard) [inline main/models.py]: Unnecessary blank line

File: main/models.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    author = models.ForeignKey(UserProfile)
+    entry = models.ForeignKey(Entry)
+    content = models.TextField()
+    timestamp = models.DateTimeField(auto_now_add=True)
+
+
+class Subscription(models.Model):
+
```
Comment (artpard) [inline main/models.py]: Unnecessary blank line

File: main/models.py [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    author = models.ForeignKey(UserProfile)
+    entry = models.ForeignKey(Entry)
+
+
+class Report(models.Model):
+
```
Comment (artpard) [inline main/models.py]: Unnecessary blank line

File: main/models.py [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+    entry = models.ForeignKey(Entry)
+    description = models.TextField()
+    timestamp = models.DateTimeField(auto_now_add=True)
+    is_open = models.BooleanField(default=True)
+    photo = models.TextField()
+    type = models.IntegerField(choices=TYPES)
+
+
+class Places(models.Model):
+
```
Comment (artpard) [inline main/models.py]: Unnecessary blank line

File: main/models.py [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+    user = models.ForeignKey(UserProfile)
+    location = models.TextField()
+    radius = models.IntegerField()
+
+
+class EntryTags(models.Model):
+
```
Comment (artpard) [inline main/models.py]: Unnecessary blank line

**Comment by artpard** [comment]
Overall looks good, except few unnecessary blank line.

**Comment by adasikow** [comment]
Well, PyCharm 'told' me that, according to PEP8, there should be a blank line between class name and class fields, so I've added them to get rid of there annoying warnings.

**Comment by artpard** [comment]
Hmm.. I was almost 100% sure that those blank lines are PEP8 violation, but if PEP8 is ok with that blank line then I am fine with that.

**Comment by adasikow** [comment]
http://stackoverflow.com/questions/17295244/blank-line-python-pep8-best-practice-in-class-definition
Looks like it's not clearly specified in PEP8.

**Comment by adasikow** [comment]
Problem solved - I've just discovered that PyCharm accepts both conventions.

**Comment by artpard** [comment]
Looks good now.

**Pull request merged**